src/core/lib/http/httpcli_security_connector.cc

```diff
@@ -132,6 +132,10 @@ static grpc_security_status httpcli_ssl_channel_security_connector_create(
     *sc = NULL;
     return GRPC_SECURITY_ERROR;
   }
+  // We don't actually need a channel credentials object in this case,
+  // but we set it to a non-NULL address so that we don't trigger
+  // assertions in grpc_channel_security_connector_cmp().
+  c->base.channel_creds = (grpc_channel_credentials *)1;
```

Comment:
Won't you hit an issue when you ref/unref this in the connector?

Reply:
No, because we're not doing that anywhere for the httpcli_ssl_channel_security_connector.  If we had a real credentials object in this case, this line is the point at which we would be reffing it, and we'd be unreffing it in `httpcli_ssl_destroy()`.  Neither of those things are happening, so this is not an issue.